tap_mixpanel/streams.py

```diff
@@ -285,6 +287,10 @@ def sync(self, state, catalog, config, start_date):
         days_interval = int(config.get("date_window_size", "30"))
         attribution_window = int(config.get("attribution_window", "5"))
 
+        # Set request timeout to config param `request_timeout` value.
+        # If value is 0,"0","" or not passed then it set default to 300 seconds.
+        config_request_timeout = config.get('request_timeout')
+        request_timeout = config_request_timeout and float(config_request_timeout) or REQUEST_TIMEOUT
```

Comment:
Please change it to If-else to keep the consistency.

Reply:
Updated